LiveXYZ.php - Getting rid of strtolower()

CaitiffMCPE opened the pull request:
Very well written but there is something that isn't needed.

$command->getName() returns the command name based on the plugin.yml,  not the player input.
This means strtolower($command->getName()) is useless if the selected command is already lower-case in your plugin.yml.

CaitiffMCPE committed:
LiveXYZ.php - Getting rid of strtolower()

Perfect plugin but there is something that isn't needed.

$command->getName() returns the command name based on the plugin.yml,  not the player input.
This means strtolower($command->getName()) is useless if the selected command is already lower-case in your plugin.yml.

CaitiffMCPE commented:
You are a dev of genisys, you should know this

dktapps commented:
I wrote this plugin a long time ago.

CaitiffMCPE commented:
It is a simple plugin and you edited it before 6 days which is not too long

dktapps commented:
I did, to get rid of CallbackTask. This was something I missed. Thank you for contributing.

P.S. Please avoid using the web editor in future.

CaitiffMCPE commented:
but why?

dktapps commented:
The web editor is dumb and it'll screw things up. Learn to use git and edit locally.

CaitiffMCPE commented:
Well, I will learn in the future.

CaitiffMCPE commented:
This account is an incognito account by the way. Perhaps you already know who I am

CaitiffMCPE commented:
But if you do not, I won't say ;)

dktapps commented:
@XFuryMCPE ?

CaitiffMCPE commented:
No

XFuryMCPE commented:
You haven't helped me with plugins, lol.

CaitiffMCPE commented:
Oh, really? Look at the tiny PRs I have made just for you. By the way I am just about to make another one

CaitiffMCPE commented:
I had made 1 PR on your BackToSpawn plugin - probably the simplest plugin. I fixed the syntax errors and optimized it just a little bit.

I had made 2 PRs on your TierLoot then I told you to to use loops instead of copy-paste and also added that $player is already an instance of Vector3 so you do not have to do $p->getLocation().

Before 5 minutes, I created another PR on TierLoot which is #3. So tell me, how I am not helping you with your plugins?

CaitiffMCPE commented:
_**And I am not a little ### self-proclaimed dev beastie like you so yeah <3**_

XFuryMCPE commented:
You commented on my old plugins that I never had planned on editing, which were created around 6-7 months ago, looks like you're doing the same here.. I would be good without any help, thanks

dktapps commented:
Okay, enough now thanks. My notifications are already overloaded.
